build/gke-test-cluster/cluster-e2e.yml

```diff
@@ -30,8 +30,6 @@ resources:
             oauthScopes:
               - https://www.googleapis.com/auth/compute
```

Comment:
Looking at https://cloud.google.com/sdk/gcloud/reference/container/clusters/create, `gke-default` is:

```
gke-default	
https://www.googleapis.com/auth/devstorage.read_only
https://www.googleapis.com/auth/logging.write
https://www.googleapis.com/auth/monitoring
https://www.googleapis.com/auth/service.management.readonly
https://www.googleapis.com/auth/servicecontrol
https://www.googleapis.com/auth/trace.append
```

Should we use the same?

Reply:
Hello @markmandel ,

Yes, if we want to have consistent scopes in docs and yml.
By the way these two are enabled by default, if monitoring and logging API is enabled:
```
https://www.googleapis.com/auth/logging.write
https://www.googleapis.com/auth/monitoring
```
Last 3 scopes were not configured but they could be helpful.
The purpose of these oauthScopes could be found [here](https://developers.google.com/identity/protocols/googlescopes).
Let me check cluster creating with those 6 and omitting ` https://www.googleapis.com/auth/compute`.